lhotse/dataset/vad.py

```diff
@@ -0,0 +1,61 @@
+from pathlib import Path
+from math import isclose
+from typing import Dict, Optional
+
+import numpy as np
+import torch
+from torch.utils.data import Dataset
+
+from lhotse.cut import CutSet
+from lhotse.utils import Pathlike, Seconds
+
+EPS = 1e-8
+
+
+class VadDataset(Dataset):
+    """
+    The PyTorch Dataset for the voice activity detection task.
+    Returns a dict of:
+
+    .. code-block::
+
+        {
+            'features': (T x F) tensor
+            'is_voice': (T x 1) tensor
+        }
+
+    In the future, will be extended by graph supervisions.
+    """
+
+    def __init__(
+            self,
+            cuts: CutSet,
+            duration: Optional[Seconds] = 5.0,
+            root_dir: Optional[Pathlike] = None
+    ):
+        super().__init__()
+        self.cuts = cuts.cut_into_windows(duration, keep_excessive_supervisions=True).cuts
+        self.root_dir = Path(root_dir) if root_dir else None
+        self.cut_ids = list(self.cuts.keys())
```

Comment:
```suggestion
        self.cut_ids = list(self.cuts.cuts.keys())
```

(if we retain a CutSet instead of dict in `self.cuts` - although it reads a bit awkward, we probably need to add `.ids` property to the CutSet)